mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreenState.kt

```diff
@@ -1,12 +1,19 @@
 package ch.epfl.sdp.mobile.ui.play
 
+import ch.epfl.sdp.mobile.application.authentication.AuthenticatedUser
 import ch.epfl.sdp.mobile.ui.social.ChessMatch
 
 /** Interface that represents state of the PlayScreen */
 interface PlayScreenState<M : ChessMatch> {
 
-  /* Callable upon actioning button */
-  fun onNewGameClick()
+  /** The current [AuthenticatedUser] */
+  val user: AuthenticatedUser
```

Comment:
Abstraction leak. I don't think we even need access to the user information from the UI.

Reply:
[f26de1b](https://github.com/epfl-SDP/android/pull/226/commits/f26de1b2a5422ea1e8d648d72f665470fe948763)